Templates/Java/BaseJavaModel.template.tt

```diff
@@ -1,6 +1,7 @@
 <# // Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information. #>
 <#@ template debug="true" hostspecific="true" language="C#" #>
 <#@ output extension="\\" #>
+<#@ import namespace="Microsoft.Graph.ODataTemplateWriter.CodeHelpers.Java" #>
```

Comment:
reviewed. just adding this note for my tracking, and so you know it has been reviewed.